apps/examples/lw_aifw_sine_test_app/lwaifw_sine_test_input.h

```diff
@@ -0,0 +1,52 @@
+/****************************************************************************
+ *
+ * Copyright 2023 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/**
+ * @file lwaifw_sine_test_input.h
+ * @brief Model input
+ */
+
+#ifndef __lwaifw_sine_test_input_h__
+#define __lwaifw_sine_test_input_h__
+
+#define NO_OF_ROWS_TO_READ 20
+#define NO_OF_SENSORS 1
+
+float gDataValues[NO_OF_ROWS_TO_READ][NO_OF_SENSORS] = {
```

Comment:
if you brought these value from tflite sample, you should put their license

Reply:
hello_tflite application generates values at runtime, I copied these values from hello_tflite output, therefore license may not be required